docs/guide/callbacks.rst

```diff
@@ -225,6 +225,29 @@ It will save the best model if ``best_model_save_path`` folder is specified and
     model = SAC("MlpPolicy", "Pendulum-v1")
     model.learn(5000, callback=eval_callback)
 
+.. _ProgressBarCallback:
+
+ProgressBarCallback
+^^^^^^^^^^^^^^^^^^^
+
+Display a progress bar with the current progress, elapsed time and estimated remaining time.
+This callback is integrated inside SB3 via the ``progress_bar`` argument of the ``learn()`` method.
+
+.. note::
+
+	This callback requires ``tqdm`` and ``rich`` packages to be installed. This is done automatically when using ``pip install stable_baselines3[extra]``
+
+
+.. code-block:: python
+
+    from stable_baselines3 import PPO
+    from stable_baselines3.common.callbacks import ProgressBarCallback
+
+    model = PPO("MlpPolicy", "Pendulum-v1", gamma=0.98, verbose=0)
```

Comment:
Do you need to specify gamma here?

Reply:
Idem for verbose

Reply:
gamma is to have a working agent, verbose is to explicitly avoid too many output in the terminal that would mess up with the progress bar

Reply:
So the progress bar is not compatible with `verbose=1`. It bothers me. I suggest to make them compatible by using 
`tqdm.write()` instead of `print()`

```python
from tqdm import tqdm

for i in tqdm(range(int(1e7))):
    if i % int(1e6) == 0:
        tqdm.write(str(i))
```

Reply:
you mean that we have to replace all the prints with write?
the rich progress bar is compatible with print, but the logger is not.

Reply:
the `external_write_mode` might be helpful there

Reply:
It is!